task/09-functions-n-closures-tasks.js

```diff
@@ -104,10 +101,19 @@ function memoize(func) {
  * retryer() => 2
  */
 function retry(func, attempts) {
-    throw new Error('Not implemented');
+  return function () {
+    while (attempts-- > 0) {
```

Comment:
You don't need 2 exit conditions. Leeav the one in the loop definition
![image](https://user-images.githubusercontent.com/7272208/88198888-dc127780-cc4c-11ea-9c57-a5f9d9967bba.png)